src/__internal__/validations/validation-icon.spec.tsx

```diff
@@ -152,48 +167,46 @@ describe("ValidationIcon", () => {
       ]);
     });
 
-    it("does not throw an error if a valid array is passed", () => {
-      jest.spyOn(global.console, "error").mockImplementation(() => {});
-      mount(
-        <ValidationIcon
-          error="Message"
-          tooltipFlipOverrides={["top", "bottom"]}
-        />
-      );
+    it("throws an error when value is not an array", () => {
+      const mockGlobal = jest
+        .spyOn(global.console, "error")
+        .mockImplementation(() => undefined);
+      const errorMessage = `The tooltipFlipOverrides prop supplied to ValidationIcon must be an array containing some or all of ["top", "bottom", "left", "right"].`;
 
-      // eslint-disable-next-line no-console
-      expect(console.error).not.toHaveBeenCalled();
-      global.console.error.mockReset();
+      expect(() => {
+        mount(
+          <ValidationIcon
+            isPartOfInput
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore:next-line
+            tooltipFlipOverrides="top"
+            error="Message"
+          />
+        );
+      }).toThrow(errorMessage);
+
+      mockGlobal.mockReset();
     });
 
-    it("throws an error if a invalid array is passed", () => {
+    it("throws an error when value is an arrya but contains an invalid element", () => {
```

Comment:
it("throws an error when value is an array but contains an invalid element", () => {